appcelerator/titanium_mobile
# [TIMOB-25910] Android: Fix Ti.UI.TableView memory leak

- Remove references to TableView rows

###### TEST CASE
```JS
const root = Ti.UI.createWindow({
        backgroundColor: 'gray'
    }),
    btn = Ti.UI.createButton({
        title: 'OPEN TABLE WINDOW'
    });

btn.addEventListener('click', () => {
    const win = Ti.UI.createWindow({
            backgroundColor: 'red'
        }),
        tableView = Ti.UI.createTableView();

    let rows = [];

    for (let i = 0; i < 64; i++) {
        const row = Ti.UI.createTableViewRow();
        const textField = Ti.UI.createTextField();

        row.add(textField);
        rows.push(row);
    }

    tableView.data = rows;

    win.add(tableView);
    win.open();
});

root.add(btn);
root.open();
```
- Open and close `TABLE WINDOW` multiple times
- Observe memory usage to not increase dramatically after GC
- Application should not crash

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-25910)
